### PR TITLE
Be/new tests no solution v2

### PR DESCRIPTION
**Pull request by beccaelenzil**
- Replaced is with ==
- Implemented functions to provide a deep copy of USER_DATA for each test to mimic fixtures
- Hardcoded data in wave 2 to scaffold the use of data from an external file (data is imported for wave 1, 3, 4, 5)
- Updated Readme to describe test_constants
- Update Readme to add note about not modifying the data

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,6 +177,12 @@ python3 play_tester.py
 
 There is some starter code provided in `play_tester.py`. This code prints the test data that is used for many of the tests. Looking closely at this data can help us think critically about the expected output for given input for each function. Then, calling each function with this data allows us to observe the **actual** output for given input. 
 
+## Test Data
+
+We will note that much of the test data for this project is provided by the file `test_constants.py`. As test data gets more and more complex, it is helpful to organize this data in its own file to enhance consistency and readability. Pytest, like many testing libraries, provide a special too for test data called **fixtures**. We will learn about fixtures later in the curriculum.
```
Comment (kaidamasaki) [inline README.md:182]: I _really_ like this note!  Setting this expectation is really great.  😄

File: tests/test_constants.py [hunk 2]
```diff
@@ -162,4 +162,18 @@
     FANTASY_2b, 
     INTRIGUE_1b,
     INTRIGUE_2b
-    ]
\ No newline at end of file
+    ]
+
+#----Functions that return clean data for each test----
```
Comment (kaidamasaki) [inline tests/test_constants.py:171]: Nice.  :smiley:  I like doing this in one place!

File: tests/test_wave_02.py [hunk 3]
```diff
@@ -1,25 +1,59 @@
 import pytest
 from viewing_party.party import *
-from tests.test_constants import USER_DATA_2
+import copy
 
 @pytest.mark.skip()
 def test_calculates_watched_average_rating():
     # Arrange
-    janes_data = USER_DATA_2
+    janes_data = {
```
Comment (kaidamasaki) [inline tests/test_wave_02.py]: I'm curious why this is all spelled out here now, instead of pulling things from test constants.

Comment (kaidamasaki) [reply tests/test_wave_02.py]: Should this be `clean_wave_2_data()`?

**Review by kaidamasaki — COMMENTED** [review]
I _really_ like these changes.  

Wave 2 looks like it's not using the helper function though but other than that I think these plus the explanation make things really clear and model good testing practices!

File: tests/test_wave_02.py (hunk 3 above)
Comment (beccaelenzil) [reply tests/test_wave_02.py]: Jasmine and I talked about having hard coded data for Wave 02, and then starting with the test_constants, but Wave_01 uses tests_costants -- so maybe that is just weird.

**Pull request merged**